packages/lua53-lpeg/build.sh

```diff
@@ -6,35 +6,38 @@ TERMUX_PKG_VERSION=1.1.0
 TERMUX_PKG_REVISION=1
```

Comment:
Please bump the revision so it's properly recognized as an update.
```suggestion
TERMUX_PKG_REVISION=2
```